package.json

```diff
@@ -94,7 +94,9 @@
 		"source-registry-storage-adapter": "github:devx-services/source-registry-storage-adapter#main",
 		"util": "^0.12.5",
 		"uuid": "^8.3.2",
-		"yaml": "^2.4.2"
+		"yaml": "^2.4.2",
+		"newrelic": "^11.0.0",
+		"@graphql-mesh/http": "0.3.26"
```

Comment:
Why do we need yaml and http packages?

Reply:
> Also this brings one more doubt if users can run mesh direclty at the root, then newrelic.cjs file will need to be added at the root

What do you mean by this? Meaning they if they use the run command outside the `init`ed folder? We don't need to solve for that. The idea is for them to use the run command from the inited folder.

Reply:
ok